sklearn/model_selection/tests/test_validation.py

```diff
@@ -134,38 +135,38 @@ def test_cross_val_score():
     for a in range(-10, 10):
         clf.a = a
         # Smoke test
-        scores = cross_val_score(clf, X, y)
```

Comment:
You are lucky that the `score` in `MockClassifier` does not depend on input. Try this with any other classifier or with `cross_val_predict` it will throw an error.